assets/css/general.css

```diff
@@ -18,6 +18,16 @@ h2 {
     transition: all 1s ease;
 }
 
+#cancelDocTranslate {
```

Comment:
Neither of these should be in `general.css`. Probably `translation.css`?

Reply:
Done. :)